.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,35 @@
+name: build_wheel
+on: [push]
+permissions: read-all
+jobs:
+  build_wheel:
+    strategy:
+      matrix:
+        python-version:
+          - 3.12.6
+    runs-on: [self-hosted, fasttext]
```

Comment:
The `fasttext` runner is being added in https://github.com/Everlaw/servers/pull/39904.